resources/conf/logback/logback-nexus.xml

```diff
@@ -1,47 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is the reason we are deleting this whole file because it's already pre-configured in Nexus 3?

Reply:
There are only 2 log configuration files for Nexus 3 when using the sonatype docker image: logback.xml and logback-access.xml.

logback-nexus.xml file in Nexus2 is now logback.xml in Nexus3 (i verified this by checking the contents of logback-nexus.xml and confirm that the contents of that also exist in logback.xml of Nexus3). Also, the contents of logback.properties (the file appender and pattern) already exists in logback.xml file. I just updated the logback.xml a little bit to include consoleAppender.

Reply:
Perfect, thanks @lykabernardo !